install_config/install/advanced_install.adoc

```diff
@@ -1936,7 +1936,7 @@ openshift_deployment_type=origin
 endif::[]
 
 # uncomment the following to enable htpasswd authentication; defaults to DenyAllPasswordIdentityProvider
-#openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true', 'challenge': 'true', 'kind': 'HTPasswdPasswordIdentityProvider', 'filename': '/etc/origin/master/htpasswd'}]
+#openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true', 'challenge': 'true', 'kind': 'HTPasswdPasswordIdentityProvider', '/etc/origin/master/htpasswd'}]
```

Comment:
Remove string '/etc/origin/master/htpasswd'

The path is hard-coded now.

Reply:
Ya, that's what I was trying to say also :smile: